src/controllers/validators/AppealReasonValidator.ts

```diff
@@ -0,0 +1,19 @@
+import { Request } from 'express';
+import { FormValidator } from './FormValidator';
+
+import { createReasonsRadioGroup } from 'app/models/components/ReasonsRadioGroup';
+import { schema } from 'app/models/fields/Reason.schema';
+import { ValidationResult } from 'app/utils/validation/ValidationResult';
+
+export class AppealReasonValidator extends FormValidator {
+    constructor() {
+        super(schema);
+    }
+
+    public async validate(request: Request): Promise<ValidationResult> {
+        const result: ValidationResult = await super.validate(request);
+
+        request.body.reasons = createReasonsRadioGroup();
```

Comment:
That seems bit like a random place to re-populate reasons list here. Validator should really be concerned with validation, nothing more.

In fact having to create separate validator which runs default validation logic and simply does totally unexpected thing is even weirder.

Reply:
For this, I'm following the pattern established in `src/controllers/MultiplePenaltiesValidator`. The code smells to me as well, but I tried many different ways to populate the reasons list on POST and none of them worked without breaking tests elsewhere in the system.

Reply:
Resolved: Removed file completely, template manipulation no longer required due to binary feature toggle.